ios/Codeforces Watcher/Application/features/users/UserViewController.swift

```diff
@@ -116,25 +131,39 @@ class UserViewController: UIViewControllerWithCross {
         }
         
         rankLabel.run {
-            $0.leadingToTrailing(of: userImage, offset: 20)
-            $0.top(to: userImage, offset: 4)
+            $0.leadingToTrailing(of: userImage, offset: 16)
+            $0.top(to: userImage, offset: 0)
         }
         
-        nameLabel.run {
-            $0.topToBottom(of: rankLabel, offset: 2)
+        handleLabel.run {
+            $0.topToBottom(of: rankLabel, offset: 0)
             $0.leading(to: rankLabel)
         }
         
-        currentRatingLabel.run {
-            $0.topToBottom(of: nameLabel, offset: 2)
+        ratingIcon.run {
+            $0.width(16)
+            $0.height(16)
+            $0.topToBottom(of: handleLabel, offset: 10)
             $0.leading(to: rankLabel)
         }
+  
+        ratingLabel.run {
+            $0.topToBottom(of: handleLabel, offset: 11)
+            $0.leftToRight(of: ratingIcon, offset: 4)
```

Comment:
leadingToTrailing?
@yevhenii-kanivets

Reply:
Yes 😉 (didn't want to complicate the stuff at first)